src/redux/rockets/rockets.js

```diff
@@ -35,6 +42,15 @@ const reducer = (state = initialState, { type, payload }) => {
   switch (type) {
     case STORE_DATA:
       return payload;
+    case TOGGLE_RESERVATION: {
+      const newState = _.map(state, (rocket) => {
+        if (rocket.id === payload) {
+          return { ...rocket, isReserved: !rocket.isReserved };
+        }
+        return rocket;
+      });
+      return newState;
+    }
```

Comment:
### [OPTIONAL] This is fine. But I have a couple observations.

1. You don't need to declare the `newState` variable. You could just return it right away.
2. You could use a ternary operator instead of an if statement. This will reduce the number of `return` keywords.

I think your code would look much cleaner that way.

Reply:
Totally agree !